### ISSUE TITLE
37,000 Games Still Herald

### ISSUE THREAD

**Issue by cedricchi** [ValveSoftware/Dota2-Gameplay#21521]
### Description

I was wondering how this ranked system still exists after 11 years. Look at this monster Valve created
https://hawk.live/posts/biggest-dota-2-fan-has-played-37000-matches-but-is-still-a-herald

**Comment by BibleEnjoyer** [comment]
Bro this isn't reddit, can u fuck off with this shit? So fucking dumb. Go post on reddit and get your 2 upvotes. This place is for bugs. You are just making it harder for the devs to fix things when they have to go through shit like this.

**Comment by cedricchi** [comment]
> Bro this isn't reddit, can u fuck off with this shit? So fucking dumb. Go post on reddit and get your 2 upvotes. This place is for bugs. You are just making it harder for the devs to fix things when they have to go through shit like this.

Do you mean the Midas bug that they put a band aid on? People who can't fix their own bug, what do you call them?

**Comment by BibleEnjoyer** [comment]
Yes, they fixed it multiple times and the game is playable again after fixes. Close this fucking report and stop being a moron.
